ivy/functional/ivy/device.py

```diff
@@ -387,6 +387,10 @@ def dev(
 # Conversions
 
 
+def get_all_native_devices():
```

Comment:
Not sure about the purpose of this function, could you please elaborate?

Reply:
was a local testing function. pushed somehow.